### DISCUSSION TITLE
How do I upgrade mason.vim itself

### DISCUSSION THREAD

**Discussion by Ryanlier** [mason-org/mason.nvim#1479] [Q&A]
Thank you for the wonderful tool. Newbie question: I ran checkhealth and got a warning if using 1.0.1 and the latest one is 1.7... I'm wondering how do I upgrade Mason.vim... there's no instructions on how to do this...

**Comment by williamboman** [comment]
Depends on how you installed it, if you use a plugin manager like packer.nvim or lazy.nvim you need to consult with their documentation.

**Reply by SupercoolBD** [reply]
ah. thank you. I use lunarvim. I'll ask there. THanks for your reply and sorry for bothering. :D
